index.js

```diff
@@ -160,9 +164,23 @@ class K8sExecutor extends Executor {
     _start(config) {
         const annotations = hoek.reach(config, 'annotations', { default: {} });
         const buildTimeout = annotations[ANNOTATE_BUILD_TIMEOUT] || this.buildTimeout;
+
+        const cpuValues = {
+            HIGH: this.highCpu,
+            LOW: this.lowCpu,
+            MICRO: this.microCpu
+        };
         const cpuConfig = annotations[CPU_RESOURCE];
-        const CPU = (cpuConfig === 'HIGH') ? this.highCpu * 1000 : this.lowCpu * 1000; // 6000 millicpu or 2000 millicpu
-        const MEMORY = (annotations[RAM_RESOURCE] === 'HIGH') ? this.highMemory : this.lowMemory;      // 12GB or 2GB
+        const CPU = (cpuConfig in cpuValues) ? cpuValues[cpuConfig] * 1000 : cpuValues.MICRO * 1000;
+
+        const memValues = {
+            HIGH: this.highMemory,
+            LOW: this.lowMemory,
+            MICRO: this.microMemory
+        };
+        const memConfig = annotations[RAM_RESOURCE];
+        const MEMORY = (memConfig in memValues) ? memValues[memConfig] : memValues.MICRO;
```

Comment:
Should we set the default setting to `MICRO`?

Reply:
I talked with @catto. This should be LOW not to break the original behavior. Thanks.